### PR TITLE
adding eonr recipe

### PR DESCRIPTION
**Pull request by tnigon**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by tnigon** [comment]
I am willing to be listed as the maintainer. Thanks!

**Comment by npavlovikj** [comment]
@tnigon , looks like you are deleting "recipes/scikits-bootstrap/*" in your PR. Can you please revert this and not delete other existing recipes?

File: recipes/eonr/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 80c31afdbbe1c0ea5579180ebb1b6cd12f6f0f3930a1d1db016810899aaa013c
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  noarch: python
+
```
Comment (marcelotrevisani) [inline recipes/eonr/meta.yaml]:
```suggestion
```

**Comment by npavlovikj** [comment]
The recipe LGTM @tnigon , thank you for contributing 🎉

**Pull request merged**